Jellyfin.Plugin.Reports/Api/Data/ReportExport.cs

```diff
@@ -15,16 +15,21 @@ public static class ReportExport
 		/// <returns> A string. </returns>
 		public static string ExportToCsv(ReportResult reportResult)
 		{
-			static void AppendRows(StringBuilder builder, List<ReportRow> rows)
+            static string EscapeText(string text)
+            {
+                string escapedText = text.Replace("\"", "\"\"", System.StringComparison.Ordinal);
+                return text.IndexOfAny(new char[4] {'"', ',', '\n', '\r'}) == -1 ? escapedText : '"' + escapedText + '"';
```

Comment:
```suggestion
                return text.IndexOfAny(new char[4] {'"', ',', '\n', '\r'}) == -1 ? escapedText : $"\"{escapedText}\"";
```

Reply:
Agreed, string interpolation is better. Change applied.